src/mastra/agents/realtime-agent.ts

```
@@ -308,15 +324,27 @@ export class RealtimeAgent extends Agent {
+    console.log('[RealtimeAgent] generateResponse called with input:', input);
+    
     // Try RAG via existing QA-agent first
     try {
       const navigator = getEngineerCafeNavigator(this.config);
       const qaAgent = navigator.getAgent('qa') as EnhancedQAAgent | undefined;
+      console.log('[RealtimeAgent] QA agent available:', !!qaAgent);
+      
       if (qaAgent && qaAgent.answerQuestion) {
+        console.log('[RealtimeAgent] Calling QA agent with input:', input);
         const qaAnswer: string = await qaAgent.answerQuestion(input);
+        console.log('[RealtimeAgent] QA agent response:', qaAnswer ? qaAnswer.substring(0, 200) + '...' : 'null/empty');
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)